modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase

```diff
@@ -90,6 +90,42 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-178087. Verify that it is possible to edit a created MDF"
+	@priority = 5
+	test CanEditMDF {
+		task ("Given I go to add a new MDF Request") {
+			JSONPRM.createMDFRequest();
+		}
+
+		task ("When the manager go to see the submitted MDF") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "manager@liferay.com",
+				userLoginFullName = "manager");
+
+			PRMNavigator.openSitePage(pageName = "MDF Requests");
+
+			PRMMDFRequest.goToMDF(campaignName = "Campaign Name");
+		}
+
+		task ("Then the manager user can edit the MDF Request") {
+			PRMMDFRequest.editMDFinformations(
+				campaignDescription = "Campaign Description Edited",
+				campaignName = "Campaign Name Edited");
+
+			PRMMDFRequest.editMDFActivity(newActivityName = "Activity Name Edited");
+		}
+
+		task ("And the user can verify that some informations were edited") {
+			PRMMDFRequest.goToMDF(campaignName = "Campaign Name Edited");
+
+			PRMMDFRequest.viewMDFActivityDetails(
+				activityName = "Activity Name Edited",
+				activityName = "Activity Name Edited");
```

Comment:
Hi @MGabriellaS, why are you passing the same variable twice? I think that it's not necessary.